misc/install.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+ENIGMA_NODE_VERSION=4.4
```

Comment:
If you just use `4` you'll now get `4.5`, where `4.4` will only give you `4.4.x`